### ISSUE TITLE
modelReady() vs this.model.on('ready', callback)

### ISSUE THREAD

**Issue by amytych** [Gapminder/vizabi#51]
They seem pretty similar, but are they exactly the same? Is there a difference between them? Can one be used in place of the other? They seem to be called multiple times but the exact number of calls differs.

I'm just trying to get a better understanding of the recent changes :-)

**Comment by arthurcamara1** [comment]
Closed by 13d644236c654a241533ffe7b27dd7ecaaeb34ac

**Comment by arthurcamara1** [comment]
Just as a quick note, `modelReady` is now called whenever there's a ready or change event, meaning that the two following would be, in practice, equivalent:

```javascript
//in the init method
this.model_binds = {
    'ready': callback,
    'change': callback
}
```
or
```javascript
modelReady: callback
```

In fact, internally `modelReady` [is called used those events](https://github.com/Gapminder/vizabi/blob/develop/src/base/component.js#L458) anyways.

So, if something should always happen whenever it's ready for the first time or when it changes, you can do it [like this](https://github.com/Gapminder/vizabi/blob/develop/src/components/_examples/display-json/display-json.js#L48), and when you want something to happen only in one case or the other, you can do it [like this](https://github.com/Gapminder/vizabi/blob/develop/src/components/_examples/bubble-chart/bubble-chart.js#L36).

Of course, `modelReady(evt)` also receives the event as a first argument, and you can always check what caused it to be called in order to execute different things.
